modules/test/playwright/pages/account-admin-web/AccountsPage.ts

```diff
@@ -3,8 +3,9 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {FrameLocator, Locator, Page} from '@playwright/test';
+import {FrameLocator, Locator, Page, expect} from '@playwright/test';
```

Comment:
Shouldn't expect come before FrameLocator?

Reply:
I think not because it's lowercase (btw sf should align them so if it doesn't complain... how are we to do it? 😄 )

Reply:
@smottal yes, you are right. Searching for "expect, FrameLocator, Locator, Page" string will yield 0 result, but there is quite a lot of result for "FrameLocator, Locator, Page, expect".